TrkReco/src/TrkTimeCalculator.cc

```diff
@@ -29,8 +29,8 @@ namespace mu2e
      _beta(          config.beta()),
      _shErr(         config.shErr()),
      _caloZOffset(   config.caloZOffset()),
-     _caloT0Offset(  config.caloT0Offset()),
-     _caloT0Err(     config.caloT0Err())
+     _caloTimeOffset(  config.caloTimeOffset()),
+     _caloTimeErr(     config.caloTimeErr())
    {}
 
 
```

Comment:
Do we still need the c'tor that takes a parameter set?

Reply:
yes, it is used by BTrk.  I don't want to invest the effort migrating those classes to validated fhicl since they will soon to become legacy.